tcproxy/tcproxy_test.go

```diff
@@ -10,7 +10,13 @@ import (
 	tcclient "github.com/taskcluster/taskcluster-client-go"
 )
 
-func TestTaskclusterProxy(t *testing.T) {
+func TestTcProxy(t *testing.T) {
```

Comment:
Why rename from `TestTaskclusterProxy` to `TestTcProxy`? I prefer the longer name.

Reply:
It duplicates another test name, and caused quite a bit of confusion trying to figure out which one had failed.  I can use another name if you'd prefer, as long as it's not a duplicate.

Reply:
That's a very good reason, thank you!